tests/zfs-tests/include/libtest.shlib

```diff
@@ -4173,6 +4173,24 @@ function get_arcstat # stat
 	esac
 }
 
+#
+# Wait for the specified arcstat to reach a non-zero plateau.
+#
+function arcstat_plateau # stat
+{
+	typeset stat=$1
+	typeset do_once=true
+
+	while $do_once || [ $stat1 -ne $stat2 ] || [ $stat2 -eq 0 ]; do
+		typeset stat1=$(get_arcstat $stat)
+		sleep 2
+		typeset stat2=$(get_arcstat $stat)
+		do_once=false
+	done
+
+	echo $stat2
```

Comment:
Did you mean to leave this echo of raw $stat2 in?

Reply:
Ah, I see, some users of arcstat_plateau use it as a value.   (Will this leave junk on stdout when they don't?)

Reply:
It seems it does, will fix.